src/app/games/game/game.component.scss

```diff
@@ -8,5 +8,7 @@ article {
     margin-left: auto;
     margin-right: auto;
     max-width: 800px;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
```

Comment:
Can probably merge these into a single rule
```suggestion
    margin: 0 1rem;
```

Reply:
Oh, is that what the default is?

Reply:
Pretty sure default is 0. Although I just realised that this is the wrong way around.
```suggestion
     margin: 1rem 0;
```

Reply:
![image](https://user-images.githubusercontent.com/73679967/145660688-49910e3a-3153-4bae-841f-588d45537835.png)
That doesn't look like zero...

Reply:
Feel free to replace 0 with some other suitable number (don't think you can inherit)